File: backend/models/log_ok.go
package models

type LogOk struct {
	LogOkId     int    `json:"log_ok_id"`
	Fecha       string `json:"fecha"`
	Modulo      string `json:"modulo"`
	Usuario     string `json:"usuario"`
	Accion      string `json:"accion"`
	Descripcion string `json:"descripcion"`
}

type LogOkDTO struct {
	FechaDesde string `json:"fecha_desde"`
	FechaHasta string `json:"fecha_hasta"`
	Modulo     string `json:"modulo"`
}

File: frontend/src/modelos/logOk.ts
export interface LogOk {
    log_ok_id: number;
    fecha: string;
    modulo: string;
    usuario: string;
    accion: string;
    descripcion: string;
}

export interface LogOkDTO {
    fecha_desde: string;
    fecha_hasta: string;
    modulo: string;
}